src/main.rs

```diff
@@ -65,6 +65,10 @@ pub struct RenderCmdOpts {
     /// Pass a regular expression to this flag to render only selected deployment(s). eg: to render only edge 'helm-templexer render --filter edge my-app.yaml'
     #[structopt(short, long)]
     filter: Option<String>,
+
+    /// Pass a command to pipe the manifest through before writing to disk, eg: 'helm-templexer render --pipe kbld my-app.yaml'
```

Comment:
```suggestion
    /// Pass one or multiple command(s) to pipe the manifest for each deployment through before writing to disk, eg: 'helm-templexer render --pipe kbld my-app.yaml'
```

Reply:
This runs `cargo run -- render --pipe='grep kind' tests/data/config_example.yaml` but this doesn't `cargo run -- render --pipe 'grep kind' tests/data/config_example.yaml`. Not sure why 🤔

Reply:
Good catch. Works for me as well with `--pipe="(...)"`. I wonder why that is 🤔 maybe one can find a hint in the Clap crate?